File: s3-inf-ppo/laboratoria/lab09/php/src/Log.php
<?php

namespace PPO\Zoo;

final class Log {

	public static function info(string $message = ""): void {
		echo $message . PHP_EOL;
	}

	public static function warning(string $message = ""): void {
		echo "[!] " . $message . PHP_EOL;
	}
	
}

File: s3-inf-ppo/laboratoria/lab09/java/src/Zoo/Log.java
package Zoo;

public class Log {

    static void info() {
        System.out.println();
    }

    public static void info(String message) {
        System.out.println(message);
    }

    static void warning(String message) {
        System.out.println("[!] "  + message);
    }

}
